scripts/singleton/get-deploy-data.js

```diff
@@ -1,19 +1,24 @@
-import Transaction from 'ethereumjs-tx'
-import ethUtils from 'ethereumjs-util'
-import fs from 'fs'
+const Transaction = require('ethereumjs-tx')
+const ethUtils = require('ethereumjs-util')
+const fs = require('fs')
 
 function toHexString (buffer) {
   return '0x' + buffer.toString('hex')
 }
 
 const bin = fs.readFileSync('singleton/singleton_RelayHub_flattened_sol_RelayHub.bin', 'ascii')
 
+// gas price for nonzero byte. 68 pre-instanbul, 16 >= istanbul
+const gtxdatanonzero = 68
+const gasLimit = 5400000
+const gasPrice = 100000000000 /// 100 gigawei
+
 const tx = new Transaction({
   nonce: 0,
-  data: '0x' + bin,
+  data: '0x' + bin + gtxdatanonzero.toString(16).padStart(64, '0'),
```

Comment:
We will have a lot of constructor parameters in the near future. I don't think manually encoding them is a viable long-term solution.